lib/pdf_fill/forms/va1010ez.rb

```diff
@@ -517,6 +519,17 @@ def merge_full_name(type)
         @form_data[type] = format_full_name(@form_data[type])
       end
 
+      def merge_address_street(address_type)
+        return unless @form_data[address_type]
+
+        address = @form_data[address_type]
+        @form_data[address_type]['street'] = combine_full_address({
+                                                                    'street' => address['street'],
+                                                                    'street2' => address['street2'],
+                                                                    'street3' => address['street3']
+                                                                  })
```

Comment:
🎨 there's no way to get it to auto-format farther left for readability, is there? maybe if this works?
```suggestion
        @form_data[address_type]['street'] = combine_full_address(
          {
            'street' => address['street'],
            'street2' => address['street2'],
            'street3' => address['street3']
          }
        )
```

Reply:
Looks like that is valid. I just pushed up the change 👍